content/applications/services/helpdesk/overview/reports.rst

```diff
@@ -1,60 +1,188 @@
+
+The :guilabel:`Customer Ratings` report is displayed in a kanban view by default, but can also be
+displayed in graph or pivot view.
+
+.. seealso::
+   - :doc:`ratings`
+
+Choosing the view and filters
+=============================
+
+The best view to display a report will depend on the what data is being measured, and how it needs
+to be grouped. See below for additional information on the available views for the
+:guilabel:`Helpdesk` reports.
+
+.. note::
+   Only one measure may be selected at a time for graphs, while pivot tables can include multiple
+   measures.
+
+Using the pivot view
+--------------------
 
-- The number of hours tickets are taking to be solved, grouped by team and ticket type.
+The *pivot* view presents data in an interactive manner. All three :guilabel:`Helpdesk` reports are
+available in the pivot view.
 
-Check if your expectations are met by *measuring* the *Time to close (hours)*. Your
-customers not only expect fast responses but they also want their issues to be handled quickly.
+Switch a report to the pivot view by selecting the icon at the top right of the screen.
 
-.. image:: reports/report_time_to_close.png
+.. image:: reports/reports-pivot-view.png
    :align: center
-   :alt: View of helpdesk ticket analysis of the hours to close by ticket type and team in
-         Odoo Helpdesk
+   :alt: View of the SLA status analysis report in Odoo Helpdesk
 
-Save filters
-============
+To add a group to a row or column, click the plus button (:guilabel:`+`) next to :guilabel:`Total`,
+and then select one of the groups. To remove one, click the minus button (:guilabel:`-`) and
+de-select.
 
-Save the filters you use the most and avoid having to reconstruct them every time they are
-needed. To do so, set the groups, filters, and measures needed. Then, go to *Favorites*.
+Using the graph view
+--------------------
 
-.. image:: reports/favorites.png
+The *graph* view presents data in either a *bar*, *line*, or *pie* chart. Switch to the graph view
+by selecting the icon at the top right of the screen. To switch between the different chart views,
+select the appropriate icon at the top left of the chart.
+
+.. tabs::
+
+   .. tab:: Bar chart
+
+       .. image:: reports/reports-bar-chart.png
+          :align: center
+          :alt: View of the SLA status analysis report in bar view
+
+   .. tab:: Line chart
+
+       .. image:: reports/reports-line-chart.png
+          :align: center
+          :alt: View of the Customer Ratings report in line view
+
+   .. tab:: Pie chart
+
+       .. image:: reports/reports-pie-chart.png
+          :align: center
+          :alt: View of the Ticket analysis report in pie chart view
+
+.. tip::
+   Both the :guilabel:`Bar Chart` and :guilabel:`Line Chart` can be viewed :guilabel:`Stacked`.
+   This presents two or more groups to appear on top of each other instead of next to each other,
+   making it easier to compare data.
```

Comment:
```suggestion
   The *Bar* and *Line* graphs can be utilize the :guilabel:`Stacked` option.
   This options presents two (or more) groups on top of each other, instead of next to each other,
   making it easier to compare data.
```